### ISSUE TITLE
Removing a group recursively fails

### ISSUE THREAD

**Issue by MatanRubin** [kronenthaler/mod-pbxproj#112]
I'm running the following script:

```
project_path = path.join(XCODE_DIR, 'my_project.xcodeproj/project.pbxproj')
project = XcodeProject.load(project_path)
project.remove_group_by_name("GroupA")
project.save()
```

And it fails with the following error:

```
File ".../lib/python2.7/site-packages/pbxproj/pbxsections/PBXGroup.py", line 69, in remove
    if subgroup is None or not subgroup.remove(recursive):
AttributeError: 'PBXFileReference' object has no attribute 'remove'
```

I took a look at the code and it looks like there's an assumption that all children of the group are groups themselves, while in practice they may be files, which do not have a "remove" attribute.

Am I missing anything?

**Comment by kronenthaler** [comment]
Thanks for the report. You're probably right, i will take a look.

**Comment by kronenthaler** [comment]
@MatanRubin could you confirm the fix on the branch `recursive-groups-removal`?

**Comment by MatanRubin** [comment]
@kronenthaler, verified the fix on `recursive-groups-removal` - looks good!
Thanks for the super quick bugfix!

**Comment by kronenthaler** [comment]
Fix released on 2.0.1. Already published on pypi.
